code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -1003,6 +1003,9 @@ GLOBAL_LIST_EMPTY(PDAs)
 		if(inserted_item)
 			to_chat(user, "<span class='warning'>There is already \a [inserted_item] in \the [src]!</span>")
 		else
+			if(C.w_class > WEIGHT_CLASS_TINY)
+				to_chat(user, "<span class='warning'>You can't fit the pen in!</span>")
```

Comment:
Should be something along the lines of "The [pen] doesn't fit!"

Reply:
@PowerfulBacon does just casting [C] to a string get the display name?